pages/14.Queue/docs.en.md

```diff
@@ -11,13 +11,11 @@ taxonomy:
 
 -----------
 
+> **_INfO:_**  Mautic 3.x users who are implementing RabbitMQ or Beanstalkd need to configure the settings directly in their local configuration file. If you are using the legacy Mautic 2.x series the steps below remains the same. 
```

Comment:
We can use the pretty styling here for a notes flash - they are documented in this PR: https://github.com/mautic/mautic-documentation/pull/162

```suggestion
> **_INFO:_**  Mautic 3.x users who are implementing RabbitMQ or Beanstalkd need to configure the settings directly in their local configuration file. If you are using the legacy Mautic 2.x series the steps below remains the same. 
```

Reply:
Can we give an example of the syntax to use?